pkg/runs/runsGet.go

```diff
@@ -179,6 +184,7 @@ func GetRunDetailsFromRasSearchRuns(runs []galasaapi.Run, apiServerUrl string) (
 func GetRunsFromRestApi(
 	runName string,
 	requestor string,
+	result string,
```

Comment:
It's odd passing a variable called result into a function ?
Perhaps resultFilter would be clearer ?